localisation/00_zuwc_united_localisation.csv

```diff
@@ -2874,6 +2867,32 @@ nick_the_decrepit;the Decrepit;;;;;;;;;;;;;x
+merge_with_champion;Merge with Champion;;;;;;;;;;;;;x
+merge_with_champion_desc;You are connected with the one who holds Frostmourne as long as a part of your soul trapped within. Though you are imprisoned in the Frozen Throne, the champion can break this ice prison and become one with you. ;;;;;;;;;;;;;x
+someone_found_frostmourne_tooltip;Someone found Frostmourne;;;;;;;;;;;;;x
+EVTDESC_WCUND_540_lich_king;[target_champion.GetBestName] couldn't reach the Frozen Throne. You no longer see [target_champion.GetHerHim]... but you feel Frostmourne is still somewhere there.;;;;;;;;;;;;;x
+EVTDESC_WCUND_540_champion;You couldn't reach the Frozen Throne. No longer you hear the voice of the Lich King, so you have no choice but to turn back.;;;;;;;;;;;;;x
+imprisoned_in_ice_tooltip;Is imprisoned within the Frozen Throne;;;;;;;;;;;;;x
+EVTDESC_WCUND_400;The [target_lich_king.GetTitle] is getting weaker, [target_lich_king.GetHerHis] power constantly seeping out of the Frozen Throne, and to prevent the Scourge from being undone an immediate action needs to be taken. As the Champion of the Scourge and wielder of Frostmourne, you must venture to Icecrown Glacier and ascend to the Frozen Throne, then claim the Helm of Domination to become a new host for the demigod and preserve control over the legions of undead lest they become a mindless horde.;;;;;;;;;;;;;x
+EVTOPTA_WCUND_400;I must save the Scourge!;;;;;;;;;;;;;x
+EVTOPTB_WCUND_400;No! I will not become the Lich King!;;;;;;;;;;;;;x
+EVTTITLE_WCUND_400;The Ascension;;;;;;;;;;;;;x
+EVTDESC_WCUND_411;Soon after what feels like a month gone by, you drop anchor in Northrend, once again. As you set foot on dry land, memories of your last ventures here come flooding back into your mind.\nYou remember when you first set down in Daggercap with your men. You remember having decided to stop at nothing to reach and destroy the Dark Lord, avenging the lives of your countrymen. The dreaded blade that made your vengeance possible cost you everything: your men, your friends, your sanity...\nIn the end, it did not matter. You were willing to pay any price to save your people, but with every brandishing of Frostmourne, you feel the guilt of your [Root.GetChampionKilledCharacter]'s death press down upon you most of all. You shove these thoughts back into the recesses of your mind, trudging ever forward into the frozen wastes.;;;;;;;;;;;;;x
```

Comment:
> mentors's

mb replace it?
It also will be strange if character is pretty old, since all its mentors may be long dead.

Reply:
Replace by what? People?

Reply:
> replace by what? People?

Then someone need to rewrite the first part of the sentence, otherwise we get this:

> **You were willing to pay any price to save your people**, but with every brandishing of Frostmourne, **you feel the guilt of your people's death press down upon you most of all**.

Reply:
Don't see workarounds, so removed this part.
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/dbbaa1bdc205f6470ab79ccd614ef6a53adb4187